tools/tertiary-analysis/scanpy/scanpy-neighbours.xml

```diff
@@ -9,7 +9,11 @@
 ln -s '${input_obj_file}' input.h5 &&
 PYTHONIOENCODING=utf-8 scanpy-neighbors
 #if $settings.default == "false"
-    --n-neighbors '${settings.n_neighbours}'
+    #if $settings.n_neighbors_file
+        --neighbors \$( cat $settings.n_neighbors_file )
```

Comment:
is the flag correct? shouldn't it be
```suggestion
        --n-neighbors \$( cat $settings.n_neighbors_file )
```
as the older one?

Reply:
D'oh- thanks for the catch